implementation.js

```diff
@@ -1,6 +1,102 @@
 'use strict';
 var ES = require('es-abstract/es6');
 var supportsDescriptors = require('define-properties').supportsDescriptors;
+var has = require('has');
+var global = require('system.global')();
+
+var parseIterable = function (iterator) {
+	var done = false;
+	var iterableResponse;
+	var tempArray = [];
+
+	if (iterator && typeof iterator.next === 'function') {
+		while (!done) {
+			iterableResponse = iterator.next();
+			if (
+				has(iterableResponse, 'value')
+				&& has(iterableResponse, 'done')
+			) {
+				if (iterableResponse.done === true) {
+					done = true;
+					break; // eslint-disable-line no-restricted-syntax
+
+				} else if (iterableResponse.done !== false) {
+					break; // eslint-disable-line no-restricted-syntax
+				}
+
+				tempArray.push(iterableResponse.value);
+			} else if (iterableResponse.done === true) {
+				done = true;
+				break; // eslint-disable-line no-restricted-syntax
+			} else {
+				break; // eslint-disable-line no-restricted-syntax
+			}
+		}
+	}
+
+	return done ? tempArray : false;
+};
+
+var hasSymbols = typeof Symbol === 'function' && typeof Symbol.iterator === 'symbol';
```

Comment:
That's because `Symbol` isn't truly polyfillable - it's a native-only feature. We should be supporting shims, but not go out of our way to support shams.